Fabric.Identity.API/scripts/Install-Discovery.ps1

```diff
@@ -0,0 +1,76 @@
+﻿param(
+    [PSCredential] $credential, 
+    [Hashtable] $configStore = @{Type = "File"; Format = "XML"; Path = "$PSScriptRoot\install.config"},
+    [switch] $quiet
+)
+
+Import-Module -Name .\Install-Discovery-Utilities.psm1 -Force
```

Comment:
Is the discovery utilities not being brought over as well?

Reply:
I have merged everything into the identity utilities.